tdp_vars_defaults/hue/hue.yml

```diff
@@ -79,13 +79,20 @@ oidcauth:
   login_redirect_url_failure: https://[hue_address]/hue/oidc_failed/
   create_users_on_login: true
   oidc_username_attribute: preferred_username
-database:
-  engine: "postgresql_psycopg2"
-  host: "{{ groups['postgresql'][0] | tosit.tdp.access_fqdn(hostvars) }}"
-  port: "{{ postgresql_server_port }}"
-  user: "{{ db_hue_user }}"
-  password: "{{ db_hue_password }}"
-  name: "{{ db_hue_name }}"
+database: >-
+  {
+  {%- if groups['postgresql'] is defined and groups['postgresql'] | length > 0 -%}
+    "engine": "postgresql_psycopg2",
+    "host": "{{ groups['postgresql'][0] | tosit.tdp.access_fqdn(hostvars) }}",
+    "port": "{{ postgresql_server_port }}",
+    "user": "{{ db_hue_user }}",
+    "password": "{{ db_hue_password }}",
+    "name": "{{ db_hue_name }}"
+  {%- else -%}
+    "engine": "sqlite3",
+    "name": "/home/hue/hue.db"
```

Comment:
`/var/lib/hue` does not exits to put the sqlite file inside ?

Reply:
It doesn't. Should I create it ?

Reply:
It works with default name "hue"